--- a/audio/audio_send_stream.cc
+++ b/audio/audio_send_stream.cc
@@ -390,7 +390,6 @@ void AudioSendStream::Stop() {
 }
 
 void AudioSendStream::SendAudioData(std::unique_ptr<AudioFrame> audio_frame) {
-  RTC_CHECK_RUNS_SERIALIZED(&audio_capture_race_checker_);
   RTC_DCHECK_GT(audio_frame->sample_rate_hz_, 0);
   TRACE_EVENT0("webrtc", "AudioSendStream::SendAudioData");
   double duration = static_cast<double>(audio_frame->samples_per_channel_) /
@@ -407,7 +406,10 @@ void AudioSendStream::SendAudioData(std::unique_ptr<AudioFrame> audio_frame) {
     MutexLock lock(&audio_level_lock_);
     audio_level_.ComputeLevel(*audio_frame, duration);
   }
-  channel_send_->ProcessAndEncodeAudio(std::move(audio_frame));
+  {
+    MutexLock lock(&audio_capture_lock_);
+    channel_send_->ProcessAndEncodeAudio(std::move(audio_frame));
+  }
 }
 
 bool AudioSendStream::SendTelephoneEvent(int payload_type,
--- a/audio/audio_send_stream.h
+++ b/audio/audio_send_stream.h
@@ -164,7 +164,7 @@ class AudioSendStream final : public webrtc::AudioSendStream,
   const Environment env_;
 
   SequenceChecker worker_thread_checker_;
-  RaceChecker audio_capture_race_checker_;
+  mutable Mutex audio_capture_lock_;
 
   const bool allocate_audio_without_feedback_;
   const bool force_no_audio_feedback_ = allocate_audio_without_feedback_;
